v2/internal/controllers/crd_apimanagement_20220801_test.go

```diff
@@ -469,6 +491,162 @@ func APIM_Api_CRUD(tc *testcommon.KubePerTestContext, service client.Object) {
 	tc.T.Log("cleaning up api")
 }
 
+func APIM_AuthorizationProvider_CRUD(tc *testcommon.KubePerTestContext, service client.Object, secretsMap *genruntime.SecretMapReference) {
+
```

Comment:
minor: other tests in this file don't have a newline as the first line in these functions. Might be worth removign this to match that.